src/test/integration/things-test.js

```diff
@@ -653,12 +651,10 @@ describe('things/', function() {
     expect(res).toBeTruthy();
 
     expect(messages[0].messageType).toEqual(Constants.EVENT);
-    expect(messages[0].data.name).toEqual(eventAFirst.name);
-    expect(messages[0].data.description).toEqual(eventAFirst.description);
+    expect(messages[0].data).toHaveProperty(eventAFirst.name);
```

Comment:
This should also verify the event's data/description

Reply:
Events no longer include a description, but since I didn't update the calls to `new Event()`, what was previously the description is now the data. I'll update.